consumption-interface/Release_Notes_1_0_0.md

```diff
@@ -0,0 +1,15 @@
+Known issues and limitations of the Consumption Interface Supervisor Service
+
+LCI : Local Consumption Interface
+
+- LCI is not supported on Multi-Zone Supervisors.
+
+- After installing the service into a supervisor, you should expect a delay while the plugin is installed. If you do not see the plugin listed in the Resources tab under Workload Management -> Namespaces -> your namespace, please wait and then refresh the page.
+
+- After installing the service into a supervisor, you will need to reload the web page or login in a new window to allow the plugin to load properly. Failure to do so may result in seeing a 502 error returned on the UI.
```

Comment:
Should we also add an action item, where admin can restart the vsphere-ui service via VC cli?
`vmon-cli -r vsphere-ui`

Reply:
Easier to just reload the browser, isn't it?

Reply:
not always atleast in my experience, but restarting the service has always worked.